### ISSUE TITLE
About model miccai2018_10_02_init1.h5

### ISSUE THREAD

**Issue by Candyeeee** [voxelmorph/voxelmorph#21]
dear author, I have a question about the model miccai2018_10_02_init1.h5. When I train the train_miccai2018.py file, there was a parameter 'load_model_file'. You explain the parameter 'optional h5 model file to initialize with'. Thus, when I load the model miccai2018_10_02_init1.h5, I found the trained model performed worse when the training epoch increased.
I want to know what the init model miccai2018_10_02_init1.h5 is and why it can be used for initializing the network.

**Comment by adalca** [comment]
`miccai2018_10_02_init1.h5` holds the weights of the resulting model from our training on T1 MR images described in the MICCAI paper (scaled to [0, 1]). The 10 and 0.2 refer to the lambda and sigma model parameters.

What images are you training on? And when you say model performed worse, by what measure?

**Comment by yellowgardenia** [comment]
Dear author,

In your code, lambda and sigma in the loss function were given a default setting 0.02 and 10 , respectively. But in your MICCAI paper, I found that you suggested different values for the two parameters. So I'm wondering that which one should I adopted to initializing the parameters?

**Comment by adalca** [comment]
I recommend using the new implementation and default settings (for data normalized in [0, 1])

You can still use the parameters given in the paper, but you'd have to use the older implementation (see `use_miccai_int` option in networks), but the newer implementation has a more intuitive meaning to the parameters.

**Comment by yellowgardenia** [comment]
Thanks for your suggestion.

In the old version:  λ = 10000, equivalent to a diagonal standard deviation of 1 voxel for φz. If I wanna to perform the experiment with my own training data, then how should I determine the optimal lamda and sigma with the new implementation?

**Comment by adalca** [comment]
I'm not sure of the exact equivalence of old_lambda of 10000 because some small implementation details changed (e.g. how we integrate). 

What is your data? If your data is close to 1mm isotropic and normalized to [0, 1], I suspect our defaults are a good starting point. I would recommend you train a network with those, and if you'd like to find the best parameter, then fine-tune by varying one of them (e.g. lambda). We just launch a few choices (e.g. 5) -- since it's fine tuning, it shouldn't take that long. We choose the best one based on our validation set.

**Comment by yellowgardenia** [comment]
Thanks! I'll try the settings with your recommendation. And I plan to train your network with the LONI data
